resources/uaa-activator/templates/job.yaml

```diff
@@ -0,0 +1,62 @@
+# use job to have a backoff limit for retires (totally new pod will be created)
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "uaa-activator.fullname" . }}
+  labels:
+    {{- include "uaa-activator.labels" . | nindent 4 }}
+  annotations:
+    # Using post-install hook as Helm is not waiting for Jobs to be completed when they are part of the release
+    # https://github.com/helm/helm/blob/dev-v2/cmd/helm/install.go#L218
+    "helm.sh/hook": post-install,post-upgrade
```

Comment:
Is there any reason why this label has quotation marks?

Reply:
fixed